### PR TITLE
Add openstack-operator kuttl testing to CI

### PR DESCRIPTION
**Pull request by jparoly**

### CODE REVIEW

File: ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-main.yaml
Comment (viroel) [inline ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-main.yaml:84]: in order to run kuttl workflow, you also need to add:
```
- build_args:
  - name: KUTTL_VERSION
    value: v0.15.0
  dockerfile_literal: |
    FROM openstack-operator-ci-build-deploy
    ARG KUTTL_VERSION=v0.15.0
    RUN curl -s -L "https://github.com/kudobuilder/kuttl/releases/download/${KUTTL_VERSION}/kubectl-kuttl_${KUTTL_VERSION#v}_linux_x86_64" -o kubectl-kuttl
    RUN chmod +x ./kubectl-kuttl
    RUN mv ./kubectl-kuttl /usr/local/bin
    RUN yum install -y skopeo
  from: openstack-operator-ci-build-deploy
  to: openstack-operator-ci-build-deploy-kutt
```
to build the container image needed for kuttl run

**Comment by viroel** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl

**Comment by cescgina** [comment]
Judging from the failure, I think we'll need to set `export NETWORK_ISOLATION=false` in the kuttl step, as we do in the deploy one

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl

File: ci-operator/jobs/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-main-presubmits.yaml
Comment (cescgina) [inline ci-operator/jobs/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-main-presubmits.yaml:110]: Minor thing, but we should disable the `build-deploy` job

Comment (jparoly) [reply ci-operator/jobs/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-main-presubmits.yaml:110]: @cescgina How do we disable it?

Comment (cescgina) [reply ci-operator/jobs/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-main-presubmits.yaml:110]: Set `always_run: false` in that job

Comment (jparoly) [reply ci-operator/jobs/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-main-presubmits.yaml:110]: thats what i thought! thanks!

**Review by cescgina — COMMENTED** [review]
lgtm

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl

**Comment by jparoly** [comment]
Looks ready to merge.  Please if you have access, merge it.

**Comment by cescgina** [comment]
> Looks ready to merge. Please if you have access, merge it.

Let's wait until above patch is merged and we get a green run, as right now the job is broken

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl

**Comment by cescgina** [comment]
Need a new fix for telemetry changes: https://github.com/openstack-k8s-operators/openstack-operator/pull/348

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl

**Comment by jparoly** [comment]
supported by: https://github.com/openstack-k8s-operators/openstack-operator/pull/368

**Comment by jparoly** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl

**Comment by jparoly** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-openstack-operator-main-openstack-operator-build-deploy-kuttl

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**
